### DISCUSSION TITLE
Low alignment rate with takara-human-rna-tcr-umi-smarter-v2 preset

### DISCUSSION THREAD

**Discussion by renee9904** [milaboratory/mixcr#1862] [Q&A] [answered]
Hello, 

I've tried to run MiXCR analysis for TCR sequencing data (https://www.ebi.ac.uk/ena/browser/view/ERR12340490?dataType=RUN) with the preset takara-human-rna-tcr-umi-smarter-v2. My app version is  4.7.0. But the alignment rate is very low.

I used the command `mixcr analyze takara-human-rna-tcr-umi-smarter-v2 -s hsa --rna --floating-left-alignment-boundary FR1Begin --assemble-clonotypes-by VDJRegion --floating-right-alignment-boundary R1_fastq.gz R2_fastq.gz result`. I've checked the reads quality using fastqc, it looked fine. I wanted to know how to improve the alignment rate with modifying the parameter, and whether the result is acceptable for further analysis.

Thank you in advance for any suggestions.

Here is the alignment reports:
[ERR12340490.align.report.txt](https://github.com/user-attachments/files/17977287/ERR12340490.align.report.txt)
[ERR12340490.assemble.report.txt](https://github.com/user-attachments/files/17977309/ERR12340490.assemble.report.txt)
[ERR12340490.refine.report.json](https://github.com/user-attachments/files/17977333/ERR12340490.refine.report.json)

**Comment by mizraelson** [comment, marked as answer]
Hi,
There is not much to do in this case, as the reason lies within the data itself. There are a lot of reads that supposedly come from some DNA contamination.

For example, here is one of the read alignments:

<img width="1476" alt="image" src="https://github.com/user-attachments/assets/7de6d78a-86f2-47db-937b-980848d66c11">

BLAST shows it is a DNA fragment with two J genes:
<img width="1228" alt="image" src="https://github.com/user-attachments/assets/cbe463f2-222e-4aee-8592-080f4ee29e71">

Half of the data is full of these artifacts; therefore, the alignment rate is low.

Also, in your command, you can just run:

```
mixcr analyze takara-human-rna-tcr-umi-smarter-v2 \
R1_fastq.gz \
R2_fastq.gz \
result
```

There is no need for any additional parameters.

**Reply by renee9904** [reply]
Thanks for your explanation 👍
